modules/docs/mdx/welcomePage.tsx

```diff
@@ -0,0 +1,120 @@
+            marginBottom="m"
+            style={{lineHeight: '3vw'}}
+          >
+            Canvas Kit
+          </Text>
+          <Text
+            typeLevel="body.large"
+            fontSize="3vw"
+            style={{lineHeight: '2vw'}}
+            color={colors.frenchVanilla100}
+          >
+            v{version}
+          </Text>
+        </Flex>
+        <img src={headerImage} alt="test" style={{width: '100%', height: 'auto'}} />
+      </Box>
+      <Text typeLevel="body.medium">
+        {' '}
+        This project provides a set of components for the Workday Canvas Design System that can be
+        used to implement user experiences consistent with Workday Design Principles.
+      </Text>
+      <Heading size="medium" borderBottom={`1px solid ${colors.blueberry200}`} paddingBottom="xxs">
+        Quick Links
+      </Heading>
+      <Grid
+        gridAutoColumns="auto"
+        gridTemplateColumns="repeat(auto-fill, minmax(250px, 1fr))"
+        gridGap="s"
+      >
+        <Grid as={Card} gridTemplateRows="1fr 1fr 2fr 1fr" depth="none" display="grid">
+          <SystemIcon color={colors.cantaloupe400} icon={rocketIcon} size={60}></SystemIcon>
+          <Card.Heading>Getting Started</Card.Heading>
+          <Card.Body>
+            <Text>For all things getting started including helpful guides and docs.</Text>
+          </Card.Body>
+          <Grid.Item
+            as={Hyperlink}
+            alignSelf="end"
+            marginTop="xs"
+            href="/?path=/docs/guides-getting-started--page"
+          >
+            {' '}
+            Getting Started Guide
+          </Grid.Item>
+        </Grid>
+        <Grid as={Card} gridTemplateRows="1fr 1fr 2fr 1fr" depth="none" display="grid">
+          <SystemIcon color={colors.blueberry400} icon={tokensIcon} size={60}></SystemIcon>
+          <Card.Heading>Tokens</Card.Heading>
+          <Card.Body>
+            <Text>
+              Tokens are the smallest pieces of our Design System with the primary function of
+              storing UI information.
+            </Text>
+          </Card.Body>
+          <Grid.Item
+            alignSelf="end"
+            as={Hyperlink}
+            marginTop="xs"
+            href="/?path=/docs/tokens--depth"
+          >
+            {' '}
+            View Our Tokens
+          </Grid.Item>
+        </Grid>
+        <Grid as={Card} gridTemplateRows="1fr 1fr 2fr 1fr" depth="none" display="grid">
+          <SystemIcon color={colors.greenApple400} icon={shapesIcon} size={60}></SystemIcon>
+          <Card.Heading>Assets</Card.Heading>
+          <Card.Body>
+            <Text>
+              Assets are graphics which help communicate meaning or highlight areas of interaction
+              to our users.
+            </Text>
+          </Card.Body>
+          <Grid.Item
+            as={Hyperlink}
+            alignSelf="end"
+            marginTop="xs"
+            href="/?path=/docs/assets-icons--icon-list"
```

Comment:
same here if we had an overview with all the icon lists

Reply:
I like that!